languages/javascript/keywords/const.md

```diff
@@ -4,9 +4,19 @@ The `const` keyword is used to create a read-only reference to a value.
 ```javascript
 const NAME = "Bob";
 ```
-The value it holds is not immutable, but the variable identifier cannot be reassigned:
+The value it holds is still [mutable](https://developer.mozilla.org/en-US/docs/Glossary/Mutable), but the variable identifier cannot be reassigned:
```

Comment:
I meant linking to OUR reference about mutability.  I do however really like this resource, so we should add this resource to OUR reference about mutability.

We do not yet have a language specific document on (im)mutability, and [here](https://github.com/exercism/v3/tree/1acc146c13dcc5268c30ab40d7e76f35285ef567/languages/javascript/reference#functional-concepts) it refers to the generic one. I recomend creating a new file [here](https://github.com/exercism/v3/tree/1acc146c13dcc5268c30ab40d7e76f35285ef567/languages/javascript/reference/info), and linking to both [`/reference/immutability`](https://github.com/exercism/v3/blob/master/reference/concepts/immutability.md) and [mdn](https://developer.mozilla.org/en-US/docs/Glossary/Mutable).

**Note:** I am specifically linking this tree from a branch, because you should base it off that branch. The files on `master` are not in the right location.